doc/ref_arch/openstack/chapters/chapter02.rst

```diff
@@ -597,24 +599,24 @@ System Hardening Requirements
        example, block, object and file storage, with access to encryption
        keys restricted based on a need to know
        (`Controlled Access Based on the Need to Know
-       <https://www.cisecurity.org/controls/controlled-access-based-on-the-need-to-know/>`__).
+       <https://www.cisecurity.org/controls/controlled-access-based-on-the-need-to-know/>`__)
      - :ref:`ref_arch/openstack/chapters/chapter06:\
        confidentiality and integrity`
    * - sec.gen.012
      - Hardening
      - The Operator **must** ensure that only authorised actors have physical
-       access to the underlying infrastructure.
-     - This requirement's verification goes beyond Anuket testing scope
+       access to the underlying infrastructure
+     - This requirement's verification must be part of the organisation security process
```

Comment:
'organisation security process' reads a little odd, should this rather be 'organisation's security process'?

Reply:
@ildikov Thanks, I will modify